go/webtest/webtest_test.go

```diff
@@ -40,7 +40,7 @@ func TestProvisionBrowser_NoCaps(t *testing.T) {
 	}
 
 	url, err := wd.CurrentURL()
-	if err != nil {
+	if err == nil {
```

Comment:
If you intended to flip this, you should change the t.Error below to print out a reason why err == nil is a problem or else it will print "nil" and bail